README.md

```diff
@@ -17,7 +17,7 @@ $(token) is a connection token generated by the proxy while connecting to the
 runtime.
 
 The shim communicates with the proxy using the protocol defined here:
```

Comment:
It would be best to avoid the use of "here" with a link following. An embedded link would be ideal, could use something like this:
The shim communicates with the proxy using the protocol defined in the `package api <https://godoc.org/github.com/clearcontainers/proxy/api>`_.